src/develop/blend.h

```diff
@@ -25,7 +25,7 @@
 #include "dtgtk/gradientslider.h"
 #include "gui/color_picker_proxy.h"
 
-#define DEVELOP_BLEND_VERSION (10)
+#define DEVELOP_BLEND_VERSION (11)
```

Comment:
that's going to conflict with #8673

Reply:
Yep, you'll need to rebase your work on top of this when merged.

Reply:
No, I mean I will lose edits. Rebasing is easy.

Reply:
Oh I see, you are already using it ! You may have to create a temp legacy_params() to upgrade your edits then.